AndroidSDKCore/src/main/java/com/leanplum/internal/VarCache.java

```diff
@@ -435,6 +451,13 @@ public static void saveDiffs() {
     } catch (JSONException e1) {
       Log.e("Error converting " + variants + " to JSON.\n" + Log.getStackTraceString(e1));
     }
+
+    if (variantDebugInfo != null) {
```

Comment:
variantDebugInfo != null will always be true,
should we check for size instead ?